src/components/alerting.tsx

```diff
@@ -1618,6 +1618,25 @@ const AlertTableRow_: React.FC<AlertTableRowProps> = ({ history, obj }) => {
     );
   }
 
+  const handleLightSpeedActions = (actions: Action[]) => {
```

Comment:
If this function got called twice, all the dropdown actions would be doubled. It feels safer to change this to a "getExtensionDropdownItems" style function that just returns the `DropdownItem`s from any extensions.

Reply:
Updated in the latest commit.